### PR TITLE
Created blog on how to raise funds using Angor

### PR DESCRIPTION
**Pull request by Aashutoshbro**
How to Raise Funds Using Angor

- Added step-by-step guide for raising funds with Angor
- Included detailed instructions for creating wallets and projects
- Provided insights on managing milestones and claiming funds

Please review this branch for the proposed changes. After aligning with your feedback, I will merge it into the main branch and submit a pull request.

### CODE REVIEW

File: src/content/posts/how-to-raise-funds/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    - Navigate to the wallet creation section.
+    - Click on "Create Wallet."
+    - Angor will automatically set up the wallet for you.
+    - Set a strong password to protect your wallet.
+
+        ![Wallet after creation](./assets/wallet-after-creation.svg)
+
+- **Back Up Recovery Phrases**: You’ll be given a set of recovery phrases. Store these securely, as they are essential for accessing your wallet if you forget your password.
+
+Taking the time to thoroughly explore the projects will help you make an informed decision about where to invest and how to raise your funds.
```
Comment (dangershony) [inline src/content/posts/how-to-raise-funds/index.md:32]: I think this text is copy paste? we making a project here not finding one to invest

File: src/content/posts/how-to-raise-funds/index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+### Step 1: Fill in Project Metadata
+
+- **Navigate to the "Create Project" Section**: Navigate to the 'Founder' section and click on 'Create Project'.
+
+- **Enter Project Metadata**:
+  - **Project Name**: Enter a clear and descriptive name for your project.
+  - **About**: Provide detailed information about your project, including its objectives and vision.
+  - **Project Website**: Enter the URL of your project’s website, if available.
+  - **Banner**: Enter the URL of your project’s banner image to make your project page visually appealing.
+  - **Nip 05 and Nip 57 (zaps)**: Fill these fields with the relevant information if applicable.
```
Comment (dangershony) [inline src/content/posts/how-to-raise-funds/index.md:47]: I think we should link here to explain what zaps are?

File: src/content/posts/how-to-raise-funds/index.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+- As a founder, once a milestone is reached, sign the transaction to spend the funds for that milestone.
+- Ensure the spending aligns with the milestone requirements and project goals.
+
+  ![Founder stage funds claim](./assets/founder-stage-claim.svg)
+
+### Claiming Funds 
+
+As a founder on Angor involves reaching and verifying milestones, providing updates, and then releasing the funds tied to those milestones through time-lock contracts. This ensures that the project progresses in a structured and accountable manner, protecting both the founder and the investors.
+
+#### 1. Monitor Project Progress
```
Comment (dangershony) [inline src/content/posts/how-to-raise-funds/index.md:98]: this are unrelated to the founder it is a bout the investor right?

File: src/content/posts/how-to-raise-funds/index.md (hunk 2 above)
Comment (Aashutoshbro) [reply src/content/posts/how-to-raise-funds/index.md:47]: Adding info about zaps would be great

File: src/content/posts/how-to-raise-funds/index.md (hunk 3 above)
Comment (Aashutoshbro) [reply src/content/posts/how-to-raise-funds/index.md:98]: Added such that being a investor is also like a founder but I will sort that out

File: src/content/posts/how-to-raise-funds/index.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+- Once a milestone is reached, navigate to your project dashboard.
+- Find the milestone that has been achieved and click on the "Claim Funds" button.
+- Sign the transaction to release the funds for that milestone.
+
+#### 3. Handling Penalties: 
+In some cases, you may need to claim funds that are subject to a penalty. Here’s how to handle that situation.
+  
+  ![Portfolio Image](./assets/invested-portfolio.svg)
+
+#### a. Claim Funds with Penalty
```
Comment (dangershony) [inline src/content/posts/how-to-raise-funds/index.md:105]: you can delete this whole section because founders can't claim funds to penalty

File: src/content/posts/how-to-raise-funds/index.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- Find the milestone that has been achieved and click on the "Claim Funds" button.
+- Sign the transaction to release the funds for that milestone.
+
+#### 3. Regular Fund Claim Process:
+
+- Navigate to the "Funds" section in your project dashboard.
+- Click on the "Claim Funds" button for the available milestone funds.
+- Sign the transaction to transfer the funds to your wallet.
+  
+  ![Portfolio Image](./assets/invested-portfolio.svg)
```
Comment (dangershony) [inline src/content/posts/how-to-raise-funds/index.md:116]: this is the image for the investor not the founder so no need to show it, instead you can show the project page with the progress bar of investor

File: src/content/posts/how-to-raise-funds/index.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+### Create a Wallet
+
+Before you can start raising funds, you need to set up a digital wallet on Angor. If you don’t have a wallet yet, create a wallet to use Angor. Follow the on-screen instructions to create your wallet.
+- Navigate to the wallet creation section.
+- Click on "Create Wallet."
+- Angor will automatically set up the wallet for you.
+- Set a strong password to protect your wallet.
+
+  ![Wallet after creation](./assets/wallet-after-creation.svg)
```
Comment (dangershony) [inline src/content/posts/how-to-raise-funds/index.md:22]: I think we also need an image here for the wallet popup (before creation) otherwise it is a bit confusing to the user they see a popup and don't know if it is correctly part of the flow

Comment (Aashutoshbro) [reply src/content/posts/how-to-raise-funds/index.md:22]: It would be more clear then

**Review by dangershony — APPROVED** [review]
